remix-gql-tailwind/app/root.tsx

```diff
@@ -57,9 +66,39 @@ export function CatchBoundary() {
   );
 }
 
+export const action: ActionFunction = async ({ request }) => {
+  await auth.logout(request, { redirectTo: '/login' });
+};
```

Comment:
Move to a resource route `/routes/logout` since the root component semantically shouldn't handle a logout action